images_config/ks-robot-slave.cfg

```diff
@@ -0,0 +1,639 @@
+deb-src http://security.ubuntu.com/ubuntu trusty-security universe
+deb http://security.ubuntu.com/ubuntu trusty-security multiverse
+deb-src http://security.ubuntu.com/ubuntu trusty-security multiverse
+
+## Uncomment the following two lines to add software from Canonical's
+## 'partner' repository.
+## This software is not part of Ubuntu, but is offered by Canonical and the
+## respective vendors as a service to Ubuntu users.
+# deb http://archive.canonical.com/ubuntu trusty partner
+# deb-src http://archive.canonical.com/ubuntu trusty partner
+
+## This software is not part of Ubuntu, but is offered by third-party
+## developers who want to ship their latest software.
+deb http://extras.ubuntu.com/ubuntu trusty main
+deb-src http://extras.ubuntu.com/ubuntu trusty main
+
+EOP
+) > /etc/apt/sources.list
+
+apt-get update
+apt-get upgrade -y
+
+
+sleep 20
+apt-get clean
+apt-get update
+aptitude install openssh-client=1:5.9p1-5ubuntu1.10
+apt-get install openssh-server -y
+sleep20
+apt-get install aptitude -y
+apt-get install ros-indigo-rosbash -y -f -y
+apt-get update
+
+apt-get install python-ros-* -y -f -y
+sleep 20
+
+apt-get clean
+apt-get update
+apt-get upgrade -y
+apt-get -f install 
+sleep20
+apt-get install ros-indigo-care-o-bot-robot -y -f -y
+sleep20
+
+rosdep init
+rosdep update
+
+
+
+echo -e "\n${green}INFO: Setup bash environment${NC}\n"
+sleep 5
+ROBOT=$(echo ${HOSTNAME%-*})
+echo $ROBOT
+if [[ "$HOSTNAME" == "$ROBOT-t"* ]]
+then
+wget https://raw.githubusercontent.com/ipa-nhg-dd/setup_cob4/master/cob-pcs/cob.bash.bashrc.t -P /etc/cob.bash.bashrc
+fi
+
+if [[ "$HOSTNAME" == "$ROBOT-h"* ]]
+then
+wget https://raw.githubusercontent.com/ipa-nhg-dd/setup_cob4/master/cob-pcs/cob.bash.bashrc.h -P /etc/cob.bash.bashrc
+fi
+if [[ "$HOSTNAME" == "$ROBOT-s"* ]]
+then
+wget https://raw.githubusercontent.com/ipa-nhg-dd/setup_cob4/master/cob-pcs/cob.bash.bashrc.s -P /etc/cob.bash.bashrc
+fi
+
+echo -e "\n${green}INFO: Setup network interfaces${NC}\n"
+sleep 5
+
+if [[ "$HOSTNAME" == "$ROBOT-t1" ]]
+then
+
+wget https://raw.githubusercontent.com/ipa-nhg-dd/setup_cob4/master/cob-pcs/networkInterfacesT1 -P /etc/network/interfaces 
+fi
+
+
+%end
+
+
+%post  
+
+
+###############sudo gpasswd -a $(whoami) docker
+
+
+echo "\n${green}INFO:Setup local ROOT user${NC}\n"
+  sleep 5
+#passwd d root
+#/usr/bin/passwd -d root
+
+echo "Defaults rootpw" >> /etc/sudoers
+rootpw robot
+#passwd root --iscrypted $1$tEHboVqc$8dkJ2OvCfg8P3z/ru2puI0
+
+
+echo "robot-local ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers
+echo "robot ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers
+
+
+
+apt-get -y -q update
+
+HOST=$(cat /etc/hostname)
+echo $HOST
+
+SERVERNAME=$(echo ${HOST%-*}-b1)
+echo $SERVERNAME
+touch /etc/auto.direct
+
+echo "/u  -fstype=nfs4    $SERVERNAME:/" >> /etc/auto.direct
+update-rc.d autofs defaults
+service autofs restart
+modprobe nfs
+###########################NTP-CONFIGURATION######################
+echo "# Specify one or more NTP servers." >> /etc/ntp.conf
```

Comment:
Please use wget also here:

`wget -O /etc/ntp.conf https://raw.githubusercontent.com/ipa320/setup_cob4/master/cob-pcs/ntp_client`

and with sed write the servername